File: 실습/3장/02_named_default_arguments/Main.java
public class Main {

    public static void greet(String name, String message) {
        System.out.println(message + ", " + name);
    }

    public static void greet(String name) {
        greet(name, "안녕하세요");
    }

    public static void main(String[] args) {
        greet("철수");
        greet("영희", "반갑습니다");
    }
}

File: 실습/3장/02_named_default_arguments/Main.kt
fun greet(
    name: String,
    message: String = "안녕하세요"
) {
    println("$message, $name")
}

fun main() {
    greet(name = "철수")
    greet(
        name = "영희",
        message = "반갑습니다"
    )
}